server/data/premisesClient.ts

```diff
@@ -1,4 +1,10 @@
-import type { DateCapacity, NewPremises, Premises, StaffMember, UpdatePremises } from '@approved-premises/api'
+import type {
+  DateCapacity,
+  NewPremises,
+  TemporaryAccommodationPremises,
```

Comment:
I tend to prefer using `TemporaryAccommodationPremises as Premises`, so it's not so verbose in the code

Reply:
Yep, good shout